lib/rules/no-bare-strings.js

```diff
@@ -147,6 +125,23 @@ function sanitizeConfigArray(allowlist = []) {
   return allowlist.filter((option) => option !== '').sort((a, b) => b.length - a.length);
 }
 
+/**
+ * Example:
+ * obj1 = { img: ['data-alt'] } }
+ * obj2 = { img: ['alt'] } }
+ * result = { img: ['data-alt', 'alt'] } }
+ */
+function mergeObjects(obj1 = {}, obj2 = {}) {
```

Comment:
@dcyriller this is a very similar solution to what you implemented with `augmentedWith`.

Reply:
nice